src/assets/stylesheets/_social_cards.scss

```diff
@@ -116,10 +116,6 @@ $blueDarken: #0085ac;
   }
 }
 
-.seo_card_input {
```

Comment:
The background now has the wrong color in properties dialogs, some light grey instead of white.

* [ ] Please adjust the bg color for dialogs, keep in mind validation messages have a different bg